docs/example.md

```diff
@@ -128,12 +148,12 @@ You may set any other connection string options that you would like to pass to t
 
 ## ![6]({{ site.baseurl }}/assets/pce-6.png) Connection Matcher
 
-(Optional) This script defines how connections are matched.
-In most cases, the default behavior works, so you don't have to include the <span style= "font-family: courier new">connection-matcher</span> section in your \*.tdr file.
+This script is optional, and defines how connections are matched.
+In most cases, the default behavior works, so you don't have to include the <span style= "font-family: courier new">connection-matcher</span> section in your TDR file.
 
 ## ![7]({{ site.baseurl }}/assets/pce-7.png) Connection Normalizer
 
-The component defines what makes up a unique connection. This can be implemented in JavaScript or directly in the xml. Writing the required attributes list in the xml is more performant, and is recommended for most connectors.
+The component defines what makes up a unique connection. This can be implemented in JavaScript or directly in the XML file. Writing the required attributes list in XML is more performant, and is recommended for most connectors.
```

Comment:
Similar to my comment about the matcher, script based normalize should be discouraged (and likely removed) due to perf implications.  I'd consider removing mention of it from the docs.

Reply:
Revised sentences to read: The component defines what makes up a unique connection. This is best implemented directly in the XML file. Writing the required attributes list in XML is more performant than using a JavaScript file, and is recommended for most connectors.